README.md

```diff
@@ -23,6 +23,18 @@ set up your dev environment for the `netlify function` to consume this API key.
 - add the GOOGLE_API_KEY to your env variables: `export GOOGLE_API_KEY='{{YOUR_API_KEY}}'`
 - run `netlify dev` from root directory
 
+# Test the pre rendered version locally
+  
+  Before being deployed, the app is pre rendered at build time, so we can quickly give a static version of the application as soon
+  as a request hits the server. Then the javascript files that will take over the app are loaded in the background.
+  Because of that the first load of the application may not behave the same as 
+  when it ran using the command `netlify dev`. 
+  
+  **It is recommended to test changes as if there were on the server** by pre rendering the application using:
+  `npx concurrently --kill-others "npm run serve:function" "NODE_ENV=production npm run prerender"
+  ` then when the build is done, serve the 
+  dist folder using `netlify dev --dir dist/ngx-darija/browser`
```

Comment:
There is already an npm script to handle that :)

Shell:
* `GOOGLE_API_KEY=AIzaSyB_xxxxxxxx NODE_ENV=production npm run serve-prerender`

PowerShell:
`$env:GOOGLE_API_KEY="AIzaSyB_xxxxxxxx"; $env:NODE_ENV="production"; npm run serve-prerender`

Reply:
Can you please confirm that `netlify build` doesn't work for you either?

Reply:
Oh I didn't see it in the README my bad. Actually netlify build worked for me but I had to create an account on netlify, then a site then link it.
This is my system info `netlify-cli/3.10.11 linux-x64 node-v12.19.0`

Reply:
Hmm, probably a Windows issue then. I'll test that on WSL and mac and see how it behaves but netlify build should be our way to go followed by netlify dev --dir that you suggested which is a very good idea to preview the build beforehead.

Thanks @Mubramaj

Reply:
@chihab ,
You might want to change NODE_ENV to something else like NGX_DARIJA_ENV because it can conflict with your node js. "setting NODE_ENV to 'development' is a common practice for NodeJS devs"

Can you try to change all NODE_ENV to NGX_DAIJA_ENV and let me know if it fixes your issue

Reply:
@chihab done, devs will need to create a netlify account and link a site to be able to run `netlify build`. Is that ok ?

Reply:
As discussed on Discord

### Build Step
* netlify build (for people having a netlify account)
* Commands below

Shell
`GOOGLE_API_KEY=AIzaSyB_xxxxxxxx NODE_ENV=production npm run serve-prerender`

PowerShell:
`$env:GOOGLE_API_KEY="AIzaSyB_xxxxxxxx"; $env:NODE_ENV="production"; npm run serve-prerender`

### Serve step

netlify dev --dir dist/.../browser